### PR TITLE
Minor issues and improvements for doing accounting

### LINKED ISSUE #450: Minor issues and improvements for doing accounting
- [x] tabjournal kid felt to tall bredere
 - [x] aksel renhold mai 2016 bank dif highlighrt ejs error B1562 tabbankaccount classes of the diff has to be without () parentesis
 - [x] tabbankaccount, fix red color on the already used journalids
 - [x] tab journal bilagsfør knappen burde matcher de på toppen og de på bunnen
 - [x] tabjournal salary match does not work
 - [x] tab bankaccount save to update the diffs, should not be nessesary
 - [x] On accountperiod comment display 0 for null values
 - [x] suggest journal id on tabbankaccount

### PR DESCRIPTION
**Pull request by agvald**
Closes #450

### CODE REVIEW

**Review by mladjo2505 — DISMISSED** [review]

File: modules/bank/view/tabbankaccount.php [hunk 1]
```diff
@@ -497,7 +498,8 @@ class="<? $v = $bank->bankvotingperiod->AmountSaldo - $bank->prevbankaccountcalc
             $EmptyHighlightCount ++;
             $BankHiglightClass ="column_empty$EmptyHighlightCount";
         } else{
-            $BankHiglightClass = "column_$row->JournalID-$row->InvoiceNumber-$row->KID";
+            $InvoiceNumber = preg_replace(array('(\()', '(\))'), array('_', '_'), $row->InvoiceNumber);
```
Comment (agvald) [inline modules/bank/view/tabbankaccount.php]: preg_replace("/[\(\)]/g", "_", $row->InvoiceNumber)

File: modules/accounting/model/accounting.class.php [hunk 2]
```diff
@@ -2365,7 +2365,7 @@ public function getAccountPlanFromOrgNumber($OrgNumber) {
 
 
     // Check if JournalID exists with a given VoucherType
-    public function checkJournalID($VoucherType, $JournalID) {
+    public function checkJournalID($VoucherType, $JournalID, $Period) {
```
Comment (agvald) [inline modules/accounting/model/accounting.class.php]: SELECT JournalID, COUNT( * ) 
FROM accountline
WHERE Period =  '2016-11'
AND Active =1
GROUP BY JournalID

File: modules/accountplan/view/record.inc [hunk 3]
```diff
@@ -247,7 +247,7 @@ if($_lib['input']->getProperty('action_accountplan_update') || $_lib['input']->g
             }
 
             // If it is a hovedbok, enable report 1 to 5 always
-            if($AccountPlanType == 'balance' || $AccountPlanType == 'result'){
+            if($AccountPlanType == 'balance' || $AccountPlanType == 'result' || $AccountPlanType == 'employee'){
```
Comment (agvald) [inline modules/accountplan/view/record.inc]: move change to line 258

File: modules/bank/view/tabbankaccount.php [hunk 4]
```diff
@@ -179,19 +181,31 @@ function display_kontoliste($conf) {
...
+      }
 
       function highlight(elementid){
         $(elementid).addClass("highlighted");
         setTimeout(function() {
           $(elementid).removeClass("highlighted");
         } , 5000);
       }
+    $(document).ready(function() {
+      $('input:submit[name=action_save_extras], button').click(function () {
```
Comment (agvald) [inline modules/bank/view/tabbankaccount.php]: remove ', button'

File: modules/bank/view/tabbankaccount.php [hunk 5]
```diff
@@ -179,19 +181,31 @@ function display_kontoliste($conf) {
...
-      $('.navigate.to').click(function(e) {
-        var element = $(e.target);
-        var targetID = element.attr('id');
-        highlight('.column_' + targetID);
-      });
+      function update_hidden_fields() {
+        var bankvotingperiod_id = "<?= $bankvotingperiod_id ?>";
+        var AmountIn  = toNumber($('#bankvotingperiod\\.AmountIn\\.' + bankvotingperiod_id).val());
+        var AmountOut = toNumber($('#bankvotingperiod\\.AmountOut\\.' + bankvotingperiod_id).val());
+
```
Comment (agvald) [inline modules/bank/view/tabbankaccount.php]: remove

File: modules/bank/view/tabjournal.php [hunk 6]
```diff
@@ -42,7 +42,7 @@ function saveScrollCookies() {
 
       $(document).ready(
         function () {
-          $('input[type="submit"], button').click(
+          $('input[type="submit" name="action_bank_update"], button').click(
```
Comment (agvald) [inline modules/bank/view/tabjournal.php]: Remove ', button'

**Review by mladjo2505 — APPROVED** [review]
All good now

**Pull request merged**